modules/ROOT/pages/afm-add-schedule.adoc

```diff
@@ -61,6 +60,20 @@ The countdown to the first run of the selected test begins.
 Specified test times correspond with your time zone.
 ====
 
+// Explanation and table with schedule interval options by location type
+
+include::_partials/afm-ref-schedule-interval.adoc[leveloffset=+1]
+
+== Scheduling Long-Running Tests
+
+Tests for a monitor that are run in public locations silently fail when the run time exceeds 120 seconds. If you need longer tests, the best solution is to use a private location, because you can run tests of long duration in private locations. 
+
+If you want to run longer-running tests in a public location, you can:
```

Comment:
```suggestion
To run longer-running tests in a public location, you can:
```

Reply:
Although I know the tenant that "If" phrases are unnecessary, I want to make an exception in this case. It is one of multiple options, so "If ..." is more clear.